core/shared/src/main/scala/scalaz/zio/ExitResult.scala

```diff
@@ -64,17 +64,6 @@ sealed abstract class ExitResult[+E, +A] extends Product with Serializable { sel
       case (_, e @ Failed(_))           => e
     }
 
-  /**
-   * Appends this result to the specified result by choosing the left-most value
-   * that does not fail.
-   *
-   * TODO: This seems to need fixing.
-   */
-  final def <>[E1, A1 >: A](that: ExitResult[E1, A1]): ExitResult[E1, A1] = self match {
```

Comment:
Glad to see this go. It was only used once in `RTS`, but now I see you got rid of that, so perhaps `orElse` is not necessary either.

Reply:
Deleted `orElse`, I used it in an earlier version and then forgot to delete. 👍